lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -9131,7 +9131,8 @@ namespace Js
                 }
                 else
                 {
-                    JS_REENTRANT(jsReentLock, obj->DeleteItem(toIndex, PropertyOperation_ThrowOnDeleteIfNotConfig));
+                    JavascriptOperators::OP_DeleteElementI(obj, JavascriptNumber::ToVar(toIndex, scriptContext), scriptContext, PropertyOperation_ThrowOnDeleteIfNotConfig);
+                    //JS_REENTRANT(jsReentLock, obj->DeleteItem(toIndex, PropertyOperation_ThrowOnDeleteIfNotConfig));
```

Comment:
Also nitpick, please delete this commented line.

Reply:
I'm moving the throw into TypedArrayBase to be aligned with other DeleteProperty/DeleteItem handlers, so there will be no change here.

---
In reply to: [173349228](https://github.com/Microsoft/ChakraCore/pull/4790#discussion_r173349228) [](ancestors = 173349228)